modules/mauth-authenticator/src/main/java/com/mdsol/mauth/AuthenticatorConfiguration.java

```diff
@@ -8,29 +8,37 @@
   public static final String REQUEST_URL_PATH = MAUTH_SECTION_HEADER + ".request_url";
   public static final String TOKEN_URL_PATH = MAUTH_SECTION_HEADER + ".token_url";
   public static final String TIME_TO_LIVE_SECONDS = MAUTH_SECTION_HEADER + ".cache.time_to_live_seconds";
+  public static final String DISABLE_MAUTH_V1= MAUTH_SECTION_HEADER + ".disable_v1";
 
   private final String baseUrl;
   private final String requestUrlPath;
   private final String securityTokensUrlPath;
   private final Long timeToLive;
+  private final boolean disableV1;
 
   public AuthenticatorConfiguration(Config config) {
     this(
         config.getString(BASE_URL_PATH),
         config.getString(REQUEST_URL_PATH),
         config.getString(TOKEN_URL_PATH),
-        config.getLong(TIME_TO_LIVE_SECONDS)
+        config.getLong(TIME_TO_LIVE_SECONDS),
+        config.hasPath(DISABLE_MAUTH_V1) || config.isEmpty() ? config.getBoolean(DISABLE_MAUTH_V1) : false
```

Comment:
just add `disable_v1` to reference.conf instead of this logic and just use `config.getBoolean(DISABLE_MAUTH_V1)`

Reply:
Updated as suggested https://github.com/mdsol/mauth-jvm-clients/pull/89/commits/c69d4044b4570262dc44715a11106302c134ac6f